app/views/admin/preview/index.html.erb

```diff
@@ -0,0 +1,111 @@
+<% content_for :extended_header do %>
+  <h1>This is a test page, so the content is fake!</h1>
+<% end %>
+
+<%= include_gon %>
+
+<%= javascript_pack_tag 'application' %>
+<%= stylesheet_link_tag "front/sites/#{@site.id}-preview.css", media: 'all', 'data-turbolinks-track': 'reload' %>
+
+<%# Here we have access to site, if we need to pass real data, like template name %>
```

Comment:
Tiny change here: the template name doesn't matter at all because only the default template can be previewed.